features/steps/utils/geometry.py

```diff
@@ -222,3 +222,30 @@ def alignment_segment_angular_difference(
     delta = abs(current_start_direction - preceding_end_direction)
 
     return delta
+
+
+def compare_with_precision(value_1, value_2, precision, comparison_operator):
+    '''Compare the value_1 with value_2 according to a comparison operator, considering a precision tolerance.
+    
+    The valid comparison operators are:
+        'equal to';
+        'not equal to';
+        'greater than';
+        'less than';
+        'greater than or equal to';
+        'less than or equal to'. 
+    '''
+    if comparison_operator == 'equal to':
+        return math.isclose(value_1, value_2, abs_tol=precision)
```

Comment:
`rel_tol` has a default value. Should we set it to `0`? https://docs.python.org/3/library/math.html#math.isclose

Reply:
From the docs:
>  rel_tol must be greater than zero.

This appears correct to me.

Reply:
The code disagrees. I really prefer to be explicit about this. If we want a relative tolerance in addition to an absolute tolerance I prefer that we write it down in the code as an act of explicitly documenting our behaviour. I think only an absolute tolerance better matches the intent of the IFC spec (even if that's very limited).

```
>>> import math
>>> math.isclose(1.0, 1.0, rel_tol=0.)
True
>>> math.isclose(1.0, 1.0, rel_tol=-1.0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: tolerances must be non-negative
```

Reply:
I see.  Implemented in 515cc490c25ee619937491a4ba1ed0b76103f20c.